### DISCUSSION TITLE
How does the DoubleML Library handle a binary dummy and a continuous variable as treatments in the PLR model?

### DISCUSSION THREAD

**Discussion by PyRSQL** [DoubleML/doubleml-for-py#256] [Q&A]
Hello DoubleML team,

I have a question about estimating the promotion price elasticity of demand. I have a binary promotion dummy and the log of price as my treatment variables, and the log of quantity as my outcome variable. I'm wondering how the binary variable is treated by the model. Does the PLR model also 'clean' the promo dummy, or does it not when it's a binary variable?" 
Thank you for your answer.

Best regards,
 
David

**Comment by SvenKlaassen** [comment]
Hello David,

generally the [PLR model](https://docs.doubleml.org/stable/guide/models.html#partially-linear-regression-model-plr) does not treat dummy treatments considerably different. The main difference is that the learner `ml_m` for 

$$m_0(X) = \mathbb{E}[D|X]$$

should be a classifier since $m_0(X)$ is a conditional probability. The score still depends on the residuals of the treatment

$$D- m(X)$$

(see [PLR Score](https://docs.doubleml.org/stable/guide/scores.html#partially-linear-regression-model-plr)).
Could you please elaborate how you have dummy variables as treatments if want to use log(price) as treatmentvariable?

Best regards,
Sven
